Code/Engine/Audio/AudioMetadataUtils.cpp

```diff
@@ -36,6 +42,41 @@ std::string GetFileExtension(const std::string& fileName)
 }
 
 //-----------------------------------------------------------------------------------
+bool IncrementPlaycount(const std::string& fileName)
+{
+    static const char* PlaycountFrameId = "PCNT";
+
+    if (fileName.find("FLAC") != std::string::npos || fileName.find("flac") != std::string::npos)
```

Comment:
Or you could also look into whether or not a case-insensitive find exists (probably not ;P)